templates/CRM/common/l10n.js.tpl

```diff
@@ -24,6 +24,7 @@
   CRM.config.ajaxPopupsEnabled = {$ajaxPopupsEnabled|@json_encode};
   CRM.config.allowAlertAutodismissal = {$allowAlertAutodismissal|@json_encode};
   CRM.config.resourceCacheCode = {$resourceCacheCode|@json_encode};
+  CRM.config.basePage = {$basePage|@json_encode}
 
   // Merge entityRef settings
   CRM.config.entityRef = $.extend({ldelim}{rdelim}, {$entityRef|@json_encode}, CRM.config.entityRef || {ldelim}{rdelim});
```

Comment:
A couple lines down from here `CRM.url` is getting initialized with `"civicrm/placeholder-url-path"` - why is that unconditionally using "civicrm" instead of the `$basePage` variable?